### ISSUE TITLE
Vue 2 - export version was not found in 'vue'

### ISSUE THREAD

**Issue by Bluskript** [justintaddei/v-wave#6]
I am trying to use this with Nuxt Vue 2, and I am getting the following error:
```
 WARN  in ./node_modules/v-wave/dist/es/index.js

"export 'version' was not found in 'vue'
```
The directive doesn't seem to work in the app.

**Comment by justintaddei** [comment]
Very peculiar. I tested with a Vue 2 app but not with nuxt for this release. I will look into this right now.

For the time being, you can use `npm i v-wave@1.1.0` to get up and running. There is not difference in the API between v1.1.0 and v1.2.0 so the upgrade will be seamless after I work this out.

Thank you for bringing this to my attention.

**Comment by Bluskript** [comment]
Yep, that worked - thanks for maintaining such an awesome library!

**Comment by justintaddei** [comment]
Fixed in v1.2.1

```sh
$ npm i v-wave@latest
```
